src/_tests_/NavBar.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+import '@testing-library/jest-dom';
+
+describe('Navbar Component', () => {
+  beforeEach(() => {
+    render(
+      <Router>
+        <Navbar />
+      </Router>,
+    );
+  });
+
```

Comment:
- [ ] Please kindly implement tests for the functions that you have in your project as stated in the requirement. 👇 

<img width="493" alt="Untitled" src="https://github.com/Uzair-Manzoor/react-capstone/assets/103386183/bb5d768c-51ff-4441-9343-f38bcbbd76a0">